frontend-student/src/components/coursePage/CommentsSection.jsx

```
@@ -222,16 +231,18 @@ export default function CommentsSection({ videoId }) {
   };
 
-  // Extracted function to render the comments list content
   const renderCommentsListContent = () => {
     if (loadingComments) {
       return (
         <div className='flex justify-center items-center p-6'>
-          <div className='animate-spin rounded-full h-10 w-10 border-b-2 border-blue-500' aria-label="Loading comments"></div>
+          <div
+            className='animate-spin rounded-full h-10 w-10 border-b-2 border-blue-500'
+            aria-label='Loading comments'
+          ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)